From f644073dc9ab0213afa9b44a9378166c400b398e Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 28 Apr 2022 17:41:48 +0200
Subject: [PATCH] hwdb: make sure "ninja update-hwdb" works on f35

let's restore compatibility with pyparsing from fedora 35, i.e.:

python3-pyparsing-2.4.7-9.fc35.noarch

(cherry picked from commit 133a0003691daafaefa378f770ae01d01931787d)

Related: #2087778
---
 hwdb.d/ids_parser.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/ids_parser.py b/hwdb.d/ids_parser.py
index 811c12559b..ed2c615508 100755
--- a/hwdb.d/ids_parser.py
+++ b/hwdb.d/ids_parser.py
@@ -20,7 +20,7 @@ COMMENTLINE = pythonStyleComment + EOL
 EMPTYLINE = LineEnd()
 text_eol = lambda name: Regex(r'[^\n]+')(name) + EOL
 
-ParserElement.set_default_whitespace_chars(' \n')
+ParserElement.setDefaultWhitespaceChars(' \n')
 
 def klass_grammar():
     klass_line = Literal('C ').suppress() + NUM2('klass') + text_eol('text')
